packages/components/src/lozenge/src/Lozenge.css

```diff
@@ -85,11 +84,12 @@
 
 /* WARNING */
 .o-ui-lozenge.o-ui-lozenge-warning {
-    background-color: var(--o-ui-bg-alias-warning-light);
+    background-color: var(--hop-decorative-option6-surface);
```

Comment:
I think we should use the warning, danger and success tokens.

![image](https://github.com/gsoft-inc/wl-orbiter/assets/361632/0602e38c-9a0b-41f2-8ee6-bd97e7c6610a)

Reply:
I did use those at first, but then I checked with Francis Benoit and he said to use the decorative tokens.

Reply:
As for the blue, I think i used option 5, wasn't sure if I should use that or option 1.. but I got Francis' approval, he said it looked good.